common/djangoapps/enrollment/urls.py

```diff
@@ -28,3 +33,8 @@
         name='courseenrollmentdetails'
     ),
 )
+
+if settings.FEATURES.get('ENABLE_BULK_ENROLLMENT_VIEW'):
```

Comment:
Done! I followed [this pattern](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/views/views.py#L153-L154)